docroot/sites/all/themes/ilr_theme/js/ilr_theme.js

```diff
@@ -404,4 +404,22 @@
     }
   };
 
```

Comment:
Is there a reason we need javascript for this? Could we use the :hover pseudo-selector, ie `.node-promo:hover a`?